src/components/Dropdown.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { PropTypes } from 'react';
+import PureComponent from 'react-pure-render/component';
+import Radium from 'radium';
+
+const containerKey = 'containerKey';
```

Comment:
This is used to identify the dropdown container so that I can retrieve the hover state from the state stored by radium. Adding a key to the container div allows me to retrieve the radium state for that div. I renamed it and capitalized it, since it is a constant that won't change.